src/stylesheets/music.scss

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
The styling makes the component look quite nice but I think you look into the styling guide in their [contributing.md ](https://github.com/SoftEng701-Group5/assignment1/blob/main/CONTRIBUTING.md) file which outlines what convention they follow. Here is a link to [BEM](https://yandex.com/dev/bem/) which is the convention that they use.